### PR TITLE
chore: switch to npm from yarn

### PR DESCRIPTION
**Pull request by JoelEinbinder**
npm is ok

### CODE REVIEW

**Review by aslushnikov — APPROVED** [review]
Thanks Joel!

File: .travis.yml [hunk 1]
```diff
@@ -6,28 +6,24 @@ addons:
...
 cache:
-  yarn: true
   directories:
     - node_modules
 # allow headful tests
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
-install:
-  - yarn install
```
Comment (paulirish) [inline .travis.yml:17]: curious: why isn't the `npm install` step necessary any longer?

File: .gitignore [hunk 2]
```diff
@@ -8,4 +8,5 @@
 *.pyc
 .vscode
 package-lock.json
```
Comment (paulirish) [inline .gitignore:10]: dont' you want to check in the package-lock.json now?

File: .travis.yml (hunk 1 above)
Comment (JoelEinbinder) [reply .travis.yml:17]: It was never necessary, but I finally read the docs.

File: .gitignore (hunk 2 above)
Comment (JoelEinbinder) [reply .gitignore:10]: Nope. With developers using separate package managers, the locks would often be ignored. Checking in the lock files proved more of a headache than security it gives over our dev-dependencies.

Comment (aslushnikov) [reply .gitignore:10]: @paulirish package-lock.json is ignored on the `npm install puppeteer` step; on the trybots, we'd like to install the same way our users do.

Comment (paulirish) [reply .gitignore:10]: k sg!

**Review by ebidel — APPROVED** [review]
Seems fine. Why do we not like yarn though?

**Comment by aslushnikov** [comment]
> Why do we not like yarn though?

@ebidel `yarn` is great, but there's some confusion on npm vs yarn usage in the docs: #1799.

We picked NPM since that's what dev team uses on a daily basis.

**Pull request merged**